app/src/main/res/values/component_colors.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="text_input_edit_text_cursor_color">@color/primary_text_color</color>
+  <color name="toolbar_color">@color/oppia_primary</color>
+  <color name="status_bar_color">@color/oppia_primary_dark</color>
```

Comment:
These still seem quite generic. I'd expect them to be part of the color palette, and then have per-activity versions like home_activity_toolbar_color.